docs/configuration.md

```diff
@@ -2670,6 +2670,9 @@ Spark uses [log4j](http://logging.apache.org/log4j/) for logging. You can config
 `log4j.properties` file in the `conf` directory. One way to start is to copy the existing
 `log4j.properties.template` located there.
 
+By default, spark adds to the MDC 2 records: `appId`, `appName` and `taskName` you can add that to your patternLayout `%X{appId}` in order to print in the logs
```

Comment:
spark -> Spark

appId, appName, taskName, so it is 3 records?

Reply:
"... print in the logs."

Reply:
typo will fix